tests/datenkompass/test_filter.py

```diff
@@ -1,24 +1,17 @@
-from pytest import MonkeyPatch
-
-import mex.extractors.datenkompass.filter as filter_module  # otherw. builtIn is shadowed
-from mex.common.models import MergedOrganization
+from mex.extractors.datenkompass.filter import filter_for_bmg
 from tests.datenkompass.mocked_item_lists import (
     mocked_bmg,
     mocked_merged_activities,
 )
 
 
-def test_filter_for_bmg(monkeypatch: MonkeyPatch) -> None:
-    def fake_get_merged_items(
-        query_string: str,  # noqa: ARG001
-        entity_type: list[str],  # noqa: ARG001
-        had_primary_source: None,  # noqa: ARG001
-    ) -> list[MergedOrganization]:
-        return mocked_bmg()
-
-    monkeypatch.setattr(filter_module, "get_merged_items", fake_get_merged_items)
+def test_filter_for_bmg() -> None:
+    bmg_ids = {bmg.identifier for bmg in mocked_bmg()}
 
-    result = filter_module.filter_for_bmg(mocked_merged_activities())
+    result = filter_for_bmg(
+        mocked_merged_activities(),  # 3 items, one without bmg to be filtered out
```

Comment:
```suggestion
        mocked_merged_activities,  # 3 items, one without bmg to be filtered out
```
could you make `mocked_merged_activities` a fixture and maybe make the comment into an assertion above?